evm_loader/lib/src/rpc/db_call_client.rs

```diff
@@ -25,10 +25,15 @@ use std::any::Any;
 pub struct CallDbClient {
     tracer_db: TracerDb,
     pub slot: u64,
```

Comment:
I suggest making `slot` field private, because it should be used only through the `Rpc` trait.

Reply:
Just to be clear: I haven't made this field public and I'm also a little bit anxious when I see public fields mixed with private ones. 🙃